tensorboard/webapp/tbdev_upload/tbdev_upload_button_component.ts

```diff
@@ -17,32 +17,13 @@ import {MatDialog} from '@angular/material/dialog';
 
 import {TbdevUploadDialogContainer} from './tbdev_upload_dialog_container';
 
-// A list of hostname values that will trigger the tbdev-upload-button button to
-// appear.
+// A list of hostname values that will trigger the button to appear.
 const LOCAL_HOSTNAMES: string[] = ['localhost', '127.0.0.1'];
```

Comment:
Btw, of course this is a separate PR, but we need to also add the ipv6 version :)